test/csuite/schema_abort/main.c

```diff
@@ -575,11 +578,18 @@ thread_ckpt_run(void *arg)
                 continue;
             }
         }
+
         /*
-         * Since this is the default, send in this string even if running without timestamps.
+         * Set the configurations. Set use_timestamps regardless of whether timestamps are in use.
+         * Set flush_tier even if tiered is not set in the program as it should be a no-op in that
+         * case. Only report that flush is in use in the program output if tiered is actually being
```

Comment:
This is interesting. When tiered storage is off, a call to `checkpoint("flush_tier=(enabled)")` succeeds.  I see how that can be convenient, but we should make a conscious decision this is what we want.  (Rather than returning some error, like EINVAL).  I don't know the "right" approach, and I think this is fine for now, but worth discussing.